tests/snuba/api/endpoints/test_organization_sessions.py

```diff
@@ -152,6 +158,13 @@ def test_invalid_query(self):
         # since its not obvious where `message` comes from.
         assert response.data == {"detail": 'Invalid query field: "message"'}
 
+    def test_illegal_query(self):
+        response = self.do_request(
+            {"statsPeriod": "1d", "field": ["sum(session)"], "query": ["issue.id:123"]}
+        )
+        assert response.status_code == 400, response.content
+        assert response.data == {"detail": 'Invalid query field: "group_id"'}
```

Comment:
This is not ideal: The provided query field is `issue.id`, but the error message mentions `group_id`. Not sure how to solve this without a larger refactor, though.

Reply:
Yes, this was a bit of a pain back when I initially wrote this code.

`get_filter` is tightly coupled to a different product, and making the code more generic was pretty much out of scope.